pkg/backend/httpstate/backend.go

```diff
@@ -816,8 +820,33 @@ func currentProjectContradictsWorkspace(stack client.StackIdentifier) bool {
 	return proj.Name.String() != stack.Project
 }
 
+type httpCreateStackOptions interface {
+	GetTeams() []string
+}
+
+// mustExtractTeams inspects the Options provided, expecting
+// they be HTTPCreateStackOptions. If so, the teams field is extracted.
+// Otherwise, this function will panic because an invalid option type
+// was provided, which is an internal bug.
+func mustExtractTeams(opts interface{}) []string {
+	var teams []string
+	if opts != nil {
+		if httpOpts, ok := opts.(httpCreateStackOptions); ok {
+			teams = httpOpts.GetTeams()
+		} else {
+			// TODO(Robbie): Remove this panic and replace it with the library equivalent.
+			panic(
+				"If non-nil Stack Create options are passed to HTTPBackend.CreateStack," +
+					"you must provide an instance of *HTTPCreateStackOptions",
```

Comment:
Are we planning on exporting HTTPCreateStackOptions?
Or will anything that matches that interface suffice here?

Reply:
I eliminated this code path almost entirely. I'm not sure the question is relevant anymore?